builtin/mainmenu/init.lua

```diff
@@ -121,9 +122,11 @@ local function init_globals()
 	})
 
 	ui.set_default("maintab")
-	check_new_version()
 	tv_main:show()
 	ui.update()
+
+	check_reinstall_mtg()
+	check_new_version()
```

Comment:
Why is this called afterwards? There might be no compatible Minetest Game version for old clients.

Reply:
`check_new_version` is asynchronous, the order doesn't really make a difference here.